packages/showcases/css/stories/components/navigation/tabs/examples/demo.html

```diff
@@ -14,8 +13,63 @@
       <li id="tab-demo-4" role="tab" aria-selected="false">
         <a href="">Tab Name</a>
       </li>
-      <span id="block-focus-trap-end" tabindex="-1"></span>
     </ul>
     <div class="vtmn-divider" role="separator"></div>
   </nav>
 </div>
+
+<script>
+  let listTabs = document.querySelectorAll('#demo-tabs-tablist li > a');
+  let nav = document.getElementById('demo-tabs-nav');
+
+  listTabs.forEach((child, index, parent) => {
+    child.addEventListener('keydown', (e) => {
```

Comment:
Again, just my personal opinion here, but also not a fan of single letter variable :) (even for e, but could be a specific case as it is widely used and understood)

Reply:
Totally agree with you, GitHub Copilot is not perfect yet ^^'